<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_vac</class>
 <widget class="QWidget" name="new_vac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add vacancy</string>
  </property>
  <widget class="QPushButton" name="ChooseSkill">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Название вакансии</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Навыки </string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateNewSkill">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый навык</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateNewVacancy">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбранные навыки</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>115</y>
     <width>41</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveSkill">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Убрать навык</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
